Makefile

```diff
@@ -0,0 +1,56 @@
+# Makefile for Harbor-arm project
+#
+# Targets:
+#
+# all: 
+
+BUILDPATH=$(CURDIR)
+HARBOR_MAKEFILE_PATH=$(BUILDPATH)/Makefile
+HARBOR_PHOTON_MAKEFILE_PATH=$(BUILDPATH)/make/photon/Makefile
+HARBOR_PHOTON_CHARTSERVER_COMPILE_PATH=$(BUILDPATH)/make/photon/chartserver/compile.sh
+HARBOR_PHOTON_REGISTRY_DOCKERFILE_PATH=$(BUILDPATH)/make/photon/registry/Dockerfile.binary
+HARBOR_PHOTON_TRIVY-ADAPTER_DOCKERFILE_PATH=$(BUILDPATH)/make/photon/trivy-adapter/Dockerfile.binary
+HARBOR_PHOTON_NOTARY_DOCKERFILE_PATH=$(BUILDPATH)/make/photon/notary/binary.Dockerfile
+HARBOR_PHOTON_PORTAL_DOCKERFILE_PATH=$(BUILDPATH)/make/photon/portal/Dockerfile
+
+SEDCMD=$(shell which sed)
+SEDCMDI=$(SEDCMD) -i
+ifeq ($(shell uname),Darwin)
+    SEDCMDI=$(SEDCMD) -i ''
+endif
+
+_update_makefile:
+	@echo "update goharbor makefile"
+	@$(SEDCMDI) 's/--rm/--rm --env CGO_ENABLED=0 --env GOOS=linux --env GOARCH=arm64/g' $(HARBOR_MAKEFILE_PATH);
+	@$(SEDCMDI) 's/$$(DOCKERBUILD)/docker buildx build --platform linux\/arm64 --progress plain --output=type=registry/g' $(HARBOR_MAKEFILE_PATH)
+
+_update_make_photon_makefile:
+	@echo "update goharbor photon makefile"
+	@$(SEDCMDI) 's/$(DOCKERCMD) build/$(DOCKERCMD) buildx build --platform linux\/arm64 --progress plain --output=type=registry/' $(HARBOR_PHOTON_MAKEFILE_PATH)
+
+_update_photon_chartserver:
```

Comment:
just use `_update_chartserver`, no need to `append` photon here.

Reply:
ok